templates/questions/index.html.ep

```diff
@@ -77,9 +78,18 @@ $(document).ready(function() {
 	<div class="newQuestionContainer">
 		<br><label for='question' class="newPostTitle">New Question</label><br>
 		<textarea placeholder= "Enter question here" class="newPostTextarea" id="newQuestionTextarea" required></textarea><br>
-		<button class='postButton' onclick="submitQuestion()">Create </button>
+		<button class='postButton' onclick="submitQuestion">Create</button>
```

Comment:
My bad!! I don't know why I deleted ( ) after onclick="submitQuestion. I am definitely not supposed to do that :(